test/parallel/test-child-process-stdin.js

```diff
@@ -34,8 +34,8 @@ cat.on('exit', common.mustCall(function(status) {
 
```

Comment:
There are two `assert.ok()` calls in this file that would be better off as `strictEqual()` calls.